src/app/CMakeLists.txt

```diff
@@ -1,18 +1,37 @@
-setup_qt4()
-
-if(USE_DRC OR USE_DRAKE)
-  use_cpp11()
+if (${DD_QT_VERSION} VERSION_GREATER "4")
+  setup_qt5()
+  include_directories(${Qt5Core_INCLUDE_DIRS})
+else()
+  setup_qt4()
 endif()
 
 find_package(PythonLibs 2.7 REQUIRED)
 find_package(PythonInterp 2.7 REQUIRED)
 find_package(PythonQt REQUIRED)
 find_package(CTKPythonConsole REQUIRED)
 find_package(OpenGL REQUIRED)
-find_package(VTK REQUIRED)
+find_package(VTK 7.1 COMPONENTS
```

Comment:
Nit: I believe it's more usual to write `REQUIRED COMPONENTS ...` than `COMPONENTS ... REQUIRED`. (My first though when I started reading this was "VTK is no longer required??"...)